src/pages/index.tsx

```diff
@@ -115,8 +117,36 @@ export function Index(props: RouteComponentProps) {
                     </Box>
                   ))}
                 </Box>
-                <Text sx={{lineHeight: '1'}}>{palette.name}</Text>
+                <Text
+                  sx={{
+                    lineHeight: '1.2',
```

Comment:
Had to do this to fix the bottom part cutting as visible in this screenshot. Please let me know if there's another way.

<img width="1165" alt="Screenshot 2022-06-22 at 7 05 58 AM" src="https://user-images.githubusercontent.com/25580776/174948631-65d38fc0-0b16-4f0d-8f87-dbed11859f44.png">